<launch>

    <!-- open realsense camera -->
    <include file="$(find realsense_ros)/launch/realsense_ros.launch">
    <arg name="sNum1"                   value="611205001943" />
    </include>

    <node pkg="soft_object_manipulator_gpr" name="soft_object_manipulator_gpr_FeatureNet_node" type="soft_object_manipulator_gpr_FeatureNet_node" output="screen">
      <param name="RADIUS" type="double" value="0.015" />
      <param name="K" type="int" value="10" />
      <param name="TOP" type="int" value="10" />
      <param name="DOWN_SAMPLE_SIZE_1" type="double" value="0.005" />
      <param name="DOWN_SAMPLE_SIZE_2" type="double" value="0.015" />
      <param name="FPS" type="int" value="5" />
      <param name="MAX_VEL" type="double" value="0.04" />
      <param name="TOLERANCE" type="double" value="0.005" />
    </node>
</launch>
